internal/cmd/local/local_install.go

```diff
@@ -107,21 +108,34 @@ func NewCmdInstall(provider k8s.Provider) *cobra.Command {
 					local.WithPortHTTP(flagPort),
 					local.WithTelemetryClient(telClient),
 					local.WithSpinner(spinner),
-					local.WithHelmChartVersion(flagChartVersion),
 				)
 				if err != nil {
 					pterm.Error.Printfln("Failed to initialize 'local' command")
 					return fmt.Errorf("could not initialize local command: %w", err)
 				}
 
+				opts := local.InstallOpts{
+					User:             flagUsername,
+					Pass:             flagPassword,
+					HelmChartVersion: flagChartVersion,
+					ValuesFile:       flagChartValuesFile,
+					Migrate:          flagMigrate,
+					Docker:           dockerClient,
+				}
+
+				if opts.HelmChartVersion == "latest" {
+					opts.HelmChartVersion = ""
+				}
+
 				if env := os.Getenv(envBasicAuthUser); env != "" {
-					flagUsername = env
+					opts.User = env
 				}
+
 				if env := os.Getenv(envBasicAuthPass); env != "" {
-					flagPassword = env
+					opts.Pass = env
```

Comment:
nit: For the string opts (that have no default, you could just assign `opts.Pass = os.GetEnv(key)` directly, since the zero-value will be empty string anyways

Reply:
They do have a default, they come from the flag definition itself
```go
cmd.Flags().StringVarP(&flagUsername, "username", "u", "airbyte", "basic auth username, can also be specified via "+envBasicAuthUser)
cmd.Flags().StringVarP(&flagPassword, "password", "p", "password", "basic auth password, can also be specified via "+envBasicAuthPass)
```